src/Collapse.js

```diff
@@ -29,6 +29,21 @@ function getDimensionValue(dimension, elem) {
   );
 }
 
+/* Check if dimension that should be calculated is height or width */
+function getDimension(dimension = 'height') {
+  return typeof dimension === 'function' ? dimension() : dimension;
+}
+
+// for testing
+function _getScrollDimensionValue(elem, computedDimension) {
+  const scroll = `scroll${computedDimension[0].toUpperCase()}${computedDimension.slice(
+    1,
+  )}`;
+  return `${elem[scroll]}px`;
+}
+// For testing purposes we should export _getScrollDimensionValue in order to stub it
+export const collapseHelpers = { _getScrollDimensionValue, getDimension };
```

Comment:
**issue:** we generally don't want to have additional exports in modules, because it screws up the "add module exports" logic we need for transparent CJS shimming.

see further comments below

Reply:
once we remove the exports, we can inline both of these functions, as both will only be used once